test/result/function/func_string_uuid.result

```diff
@@ -0,0 +1,7 @@
+create table t1(a INT,  b float);
+insert into t1 values(12124, -4213.413), (12124, -42413.409);
+SELECT length(uuid()) FROM t1;
```

Comment:
can add uuid(1) case to test if error will throw

Reply:
OK, I add a fault tolerance test